ow_gazebo_plugins/src/BalovnevModelPlugin/BalovnevModelPlugin.cpp

```diff
@@ -111,13 +118,10 @@ void BalovnevModelPlugin::onUpdate()
 // efa - ext friction angle
 double BalovnevModelPlugin::getParameterA(double x, double ifa, double efa)
 {  
-  if (x <= 0.5 * asin(sin(efa)/sin(ifa)) - efa)
-  {
+  if (x <= 0.5 * asin(sin(efa)/sin(ifa)) - efa) {
     double a = (1 - sin(ifa) * cos(2 * x)) / (1 - sin(ifa));
     return a;
-  } 
-  else 
-  {
+  } else {
```

Comment:
ROS style wants braces on their own lines

Reply:
Good catch, though we have been using "cuddled" braces since the start of the project, and "uncuddled" ones as well.   I've added a proposed change to the coding guidelines that we can review next week.

Reply:
Right you are. Although the project has not kept to this convention, which might be why it has been purposed to loosen the requirement in our own [internal code style](https://babelfish.arc.nasa.gov/confluence/display/OCEANWATERS/Coding+Style+and+Practices). I'll change this section, since the original code had it on separate lines. I am fine changing other uses where you think it'll improve readability.

Reply:
> Good catch, though we have been using "cuddled" braces since the start of the project, and "uncuddled" ones as well. I've added a proposed change to the coding guidelines that we can review next week.

Ah! I did not see this until after my own comment.